src/containers/nav/desktop/NavColumn.js

```diff
@@ -223,29 +225,32 @@ const NavColumn = ({
       <div className={cn(styles.navContent, { [styles.show]: navLoaded })}>
         <SimpleBar className={styles.scrollable}>
           {account ? (
-            <div className={styles.accountWrapper}>
-              <DynamicImage
-                wrapperClassName={styles.wrapperPhoto}
-                className={styles.dynamicPhoto}
-                onClick={goToProfile}
-                image={profileImage}
-              />
-              <div className={styles.userInfoWrapper}>
-                <div className={styles.name} onClick={goToProfile}>
-                  {name}
-                  <UserBadges
-                    userId={account.user_id}
-                    badgeSize={12}
-                    className={styles.badge}
-                  />
-                </div>
-                <div className={styles.handleContainer}>
-                  <span
-                    className={styles.handle}
-                    onClick={goToProfile}
-                  >{`@${handle}`}</span>
+            <div className={styles.userHeader}>
+              <div className={styles.accountWrapper}>
+                <DynamicImage
+                  wrapperClassName={styles.wrapperPhoto}
+                  className={styles.dynamicPhoto}
+                  onClick={goToProfile}
+                  image={profileImage}
+                />
+                <div className={styles.userInfoWrapper}>
+                  <div className={styles.name} onClick={goToProfile}>
+                    {name}
+                    <UserBadges
+                      userId={account.user_id}
+                      badgeSize={12}
+                      className={styles.badge}
+                    />
+                  </div>
+                  <div className={styles.handleContainer}>
+                    <span
+                      className={styles.handle}
+                      onClick={goToProfile}
+                    >{`@${handle}`}</span>
+                  </div>
                 </div>
               </div>
+              <NavAudio />
```

Comment:
Should showing this be wrapped in a feature flag to turn on w. rewards? I'm not sure

Reply:
yep! let's feature flag. a new one would be preferable